tests/Feature/GetUsersTest.php

```diff
@@ -1,23 +1,31 @@
 <?php
 
-use App\Services\UserDataProvider;
+use App\Infrastructure\Clients\DBClient;
 use App\Utilities\ErrorCodes;
 use Tests\TestCase;
 use Exception;
+use Mockery;
 
 class GetUsersTest extends TestCase
 {
+    protected DBClient $dbClientMock;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->dbClientMock = Mockery::mock(DBClient::class);
+        $this->app->instance(DBClient::class, $this->dbClientMock);
+    }
+
     /**
      * @test
      */
     public function GetsUsers()
```

Comment:
Estos nombres deberían ir en minúsculas
```suggestion
    public function getsUsers()
```